src/org/opendatakit/briefcase/ui/SettingsPanel.java

```diff
@@ -88,17 +89,19 @@ public void actionPerformed(ActionEvent arg0) {
     chkParallel.addActionListener(new ParallelPullToggleListener());
 
     chkTrackingConsent.setSelected(BriefcasePreferences.getBriefcaseTrackingConsentProperty());
-    chkTrackingConsent.addActionListener(new TrackingConsentToggleListener(parentWindow.briefcaseAnalytics));
+    chkTrackingConsent.addActionListener(__ -> {
+      analytics.enableTracking(chkTrackingConsent.isSelected());
+      BriefcasePreferences.setBriefcaseTrackingConsentProperty(chkTrackingConsent.isSelected());
+    });
 
     chkStorePasswordsConsent.setSelected(BriefcasePreferences.getStorePasswordsConsentProperty());
     chkStorePasswordsConsent.addActionListener(e -> {
-      if (e.getSource() == chkStorePasswordsConsent) {
-        boolean isSelected = chkStorePasswordsConsent.isSelected();
-        BriefcasePreferences.setStorePasswordsConsentProperty(isSelected);
-        if (!isSelected) {
-          PullTransferPanel.PREFERENCES.remove(BriefcasePreferences.PASSWORD);
-          PushTransferPanel.PREFERENCES.remove(BriefcasePreferences.PASSWORD);
-        }
+      boolean isSelected = chkStorePasswordsConsent.isSelected();
+      analytics.event("Settings", "Save passwords", isSelected ? "Enable" : "Disable", null);
```

Comment:
What's the rationale behind tracking this?

Reply:
I thought having some metrics on how many people are disabling or enabling this would be useful...

Reply:
I want to make sure we know what we'll do with the data before we collect it. In the case of tracking consent, that's useful because we want people to consent and so knowing when people are not consenting can help us change the messaging. 

So I don't mind that we collect this, but I do want you to defend it by explaining what action will we take based on the data we get. So yeah, if 100% save passwords what does that mean to you? And what about 50% or 0%?

Reply:
I understand the problem. I think that having metrics of a new feature we are going to release it's always useful to measure its impact. 

Also, being this password stuff somewhat controversial, the data could tell us how the users are feeling about it. My bias is that the pros of storing passwords overweight the cons, but I don't know how our users value their privacy/security. 

If all our users save passwords, we could consider securing Briefcase a little more. If none use this, we could ask on the forums for feedback (are they not using it because it's insecure? would they use it if it was secure? are they even aware that there's that possibility? etc.).

Anyway, at this point, I guess it's more about adding the ability to track into Briefcase more than the specific things we want to track from day zero... I will remove the tracking If you consider it's better that way ;)